### ISSUE TITLE
As a user, i want to see al my gaming group played games, not only the last 10

### ISSUE THREAD

**Issue by vfportero** [NemeStats/NemeStats#342]
Add a "view all" link on the bottom of the table to navigate to a view with a paged list of played games

**Comment by jakejgordon** [comment]
Also remember the poorly promoted feature where you can search for played games. It may or may not make sense to integrate with that more.

**Comment by vfportero** [comment]
you're right! The search view can solve this issue. I thought it would be easier but to achieve real pagination we must to change whole query services architecture.

Not today :(
